### ISSUE TITLE
crash on boot; ' Uncaught exception in thread "main" '

### ISSUE THREAD

**Issue by Ozzysyn** [Ladysnake/Requiem#507]
crash on boot, was able to run this mod on a previous computer of mine, and do not want to go the nuclear route of reinstalling minecraft, so i will leave this here if you can see an exact issue. apologies if it's obvious and i'm just being foolish

Uncaught exception in thread "main"
java.lang.RuntimeException: Mixin transformation of net.minecraft.client.main.Main failed
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:252)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:414)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [blabber.mixins.json:DynamicRegistryManagerMixin] from phase [DEFAULT] in config [blabber.mixins.json] from mod [blabber] FAILED during PREPARE
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinError(MixinProcessor.java:638)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinPrepareError(MixinProcessor.java:585)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.prepareConfigs(MixinProcessor.java:543)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.select(MixinProcessor.java:462)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.checkSelect(MixinProcessor.java:438)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:290)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:247)
	... 6 more
Caused by: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: @Mixin target type mismatch: net.minecraft.class_5455 is an interface in org.spongepowered.asm.mixin.transformer.MixinInfo$SubType$Standard@61d01788
	at org.spongepowered.asm.mixin.transformer.MixinInfo$SubType.validateTarget(MixinInfo.java:553)
	at org.spongepowered.asm.mixin.transformer.MixinInfo.getTargetClass(MixinInfo.java:1027)
	at org.spongepowered.asm.mixin.transformer.MixinInfo.readTargetClasses(MixinInfo.java:1008)
	at org.spongepowered.asm.mixin.transformer.MixinInfo.parseTargets(MixinInfo.java:896)
	at org.spongepowered.asm.mixin.transformer.MixinConfig.prepareMixins(MixinConfig.java:869)
	at org.spongepowered.asm.mixin.transformer.MixinConfig.prepare(MixinConfig.java:777)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.prepareConfigs(MixinProcessor.java:540)
	... 12 more

**Comment by James103** [comment]
1. Can you please post the full mod list?
2. Can you please upload any/all log files in your Minecraft instance's `.minecraft/logs` folder?

**Comment by Gaming32** [comment]
I have the same error. Here are the logs: [latest.log](https://github.com/Ladysnake/Requiem/files/8691676/latest.log)

**Comment by Derailedzack** [comment]
Requiem 2.0 beta 7, FabricAPI 0.53.0
```
[09:45:19] [main/INFO]: Loading Minecraft 1.18.2 with Fabric Loader 0.14.6
[09:45:19] [main/INFO]: Fabric is preparing JARs on first launch, this may take a few seconds...
[09:45:19] [main/WARN]: Warnings were found!
 - Mod 'Requiem' (requiem) 2.0.0-beta.7 recommends version 4.11 or later of cloth-config2, which is missing!
	 - You should install version 4.11 or later of cloth-config2 for the optimal experience.
[09:45:19] [main/INFO]: Loading 64 mods:
	- automatone 0.5.0 via requiem
	- blabber 0.1.2 via requiem
	- cardinal-components-base 4.1.2 via requiem
	- cardinal-components-entity 4.1.2 via requiem
	- cardinal-components-scoreboard 4.1.2 via requiem
	- cardinal-components-world 4.1.2 via requiem
	- chime 1.2.0-SNAPSHOT via requiem
	- fabric 0.53.0+1.18.2
	- fabric-api-base 0.4.4+d7c144a860 via fabric
	- fabric-api-lookup-api-v1 1.6.0+2373a54560 via fabric
	- fabric-biome-api-v1 8.0.4+7b0f312660 via fabric
	- fabric-blockrenderlayer-v1 1.1.12+3ac43d9560 via fabric
	- fabric-command-api-v1 1.1.9+d7c144a860 via fabric
	- fabric-commands-v0 0.2.8+b4f4f6cd60 via fabric
	- fabric-containers-v0 0.1.21+d7c144a860 via fabric
	- fabric-content-registries-v0 3.0.4+d82b939260 via fabric
	- fabric-convention-tags-v1 1.0.1+474e4f7960 via fabric
	- fabric-crash-report-info-v1 0.2.0+a6b0947860 via fabric
	- fabric-dimensions-v1 2.1.17+2540745460 via fabric
	- fabric-entity-events-v1 1.4.8+d7c144a860 via fabric
	- fabric-events-interaction-v0 0.4.19+d7c144a860 via fabric
	- fabric-events-lifecycle-v0 0.2.15+d7c144a860 via fabric
	- fabric-game-rule-api-v1 1.0.13+d7c144a860 via fabric
	- fabric-item-api-v1 1.5.0+85719ab760 via fabric
	- fabric-item-groups-v0 0.3.12+91896a4960 via fabric
	- fabric-key-binding-api-v1 1.0.12+54e5b2ec60 via fabric
	- fabric-keybindings-v0 0.2.10+b4f4f6cd60 via fabric
	- fabric-lifecycle-events-v1 2.0.3+2d83e92c60 via fabric
	- fabric-loot-tables-v1 1.0.11+d7c144a860 via fabric
	- fabric-mining-level-api-v1 2.1.1+81ea5b3c60 via fabric
	- fabric-models-v0 0.3.6+d7c144a860 via fabric
	- fabric-networking-api-v1 1.0.21+d882b91560 via fabric
	- fabric-networking-v0 0.3.8+b4f4f6cd60 via fabric
	- fabric-object-builder-api-v1 2.1.1+032c981d60 via fabric
	- fabric-particles-v1 0.2.12+526dc1ac60 via fabric
	- fabric-permissions-api-v0 0.1-SNAPSHOT via requiem
	- fabric-registry-sync-v0 0.9.8+0d9ab37260 via fabric
	- fabric-renderer-api-v1 0.4.13+d882b91560 via fabric
	- fabric-renderer-indigo 0.5.0+7faf0d8860 via fabric
	- fabric-renderer-registries-v1 3.2.11+b4f4f6cd60 via fabric
	- fabric-rendering-data-attachment-v1 0.3.7+d7c144a860 via fabric
	- fabric-rendering-fluids-v1 2.0.2+54e5b2ec60 via fabric
	- fabric-rendering-v0 1.1.13+b4f4f6cd60 via fabric
	- fabric-rendering-v1 1.10.7+54e5b2ec60 via fabric
	- fabric-resource-conditions-api-v1 2.0.3+91b7aa6660 via fabric
	- fabric-resource-loader-v0 0.5.0+2afceb2960 via fabric
	- fabric-screen-api-v1 1.0.10+d882b91560 via fabric
	- fabric-screen-handler-api-v1 1.2.0+1f6558e860 via fabric
	- fabric-textures-v0 1.0.12+3ac43d9560 via fabric
	- fabric-transfer-api-v1 1.6.1+f4563ac860 via fabric
	- fabric-transitive-access-wideners-v1 1.0.0+bd8a4a1860 via fabric
	- fabricloader 0.14.6
	- fiber 0.23.0-2 via requiem
	- fiber2cloth 3.1.1 via requiem
	- impersonate 2.4.1 via requiem
	- java 17
	- locki 0.7.0 via requiem
	- minecraft 1.18.2
	- playerabilitylib 1.5.0 via requiem
	- reach-entity-attributes 2.1.1 via requiem
	- requiem 2.0.0-beta.7
	- requiem-core 2.0.0-beta.7 via requiem
	- requiemapi 2.0.0-beta.7 via requiem
	- satin 1.7.2 via requiem
[09:45:20] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/C:/Users/zcumming-2022/AppData/Roaming/.minecraft/libraries/net/fabricmc/sponge-mixin/0.11.4+mixin.0.8.5/sponge-mixin-0.11.4+mixin.0.8.5.jar Service=Knot/Fabric Env=CLIENT
[09:45:22] [main/INFO]: Compatibility level set to JAVA_16
[09:45:22] [main/ERROR]: Mixin prepare for mod blabber failed preparing DynamicRegistryManagerMixin in blabber.mixins.json: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException @Mixin target type mismatch: net.minecraft.class_5455 is an interface in org.spongepowered.asm.mixin.transformer.MixinInfo$SubType$Standard@d59296f
org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: @Mixin target type mismatch: net.minecraft.class_5455 is an interface in org.spongepowered.asm.mixin.transformer.MixinInfo$SubType$Standard@d59296f
	at org.spongepowered.asm.mixin.transformer.MixinInfo$SubType.validateTarget(MixinInfo.java:553) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinInfo.getTargetClass(MixinInfo.java:1027) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinInfo.readTargetClasses(MixinInfo.java:1008) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinInfo.parseTargets(MixinInfo.java:896) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinConfig.prepareMixins(MixinConfig.java:869) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinConfig.prepare(MixinConfig.java:777) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.prepareConfigs(MixinProcessor.java:540) [sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.select(MixinProcessor.java:462) [sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.checkSelect(MixinProcessor.java:438) [sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:290) [sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) [sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) [sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:413) [fabric-loader-0.14.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:322) [fabric-loader-0.14.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) [fabric-loader-0.14.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:145) [fabric-loader-0.14.6.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) [?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:453) [fabric-loader-0.14.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.14.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.14.6.jar:?]
[09:45:22] [main/ERROR]: Uncaught exception in thread "main"
java.lang.RuntimeException: Mixin transformation of net.minecraft.client.main.Main failed
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:418) ~[fabric-loader-0.14.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:322) ~[fabric-loader-0.14.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.14.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:145) ~[fabric-loader-0.14.6.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:453) ~[fabric-loader-0.14.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.14.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.14.6.jar:?]
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [blabber.mixins.json:DynamicRegistryManagerMixin from mod blabber] from phase [DEFAULT] in config [blabber.mixins.json] FAILED during PREPARE
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinError(MixinProcessor.java:638) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinPrepareError(MixinProcessor.java:585) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.prepareConfigs(MixinProcessor.java:543) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.select(MixinProcessor.java:462) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.checkSelect(MixinProcessor.java:438) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:290) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:413) ~[fabric-loader-0.14.6.jar:?]
	... 7 more
Caused by: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: @Mixin target type mismatch: net.minecraft.class_5455 is an interface in org.spongepowered.asm.mixin.transformer.MixinInfo$SubType$Standard@d59296f
	at org.spongepowered.asm.mixin.transformer.MixinInfo$SubType.validateTarget(MixinInfo.java:553) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinInfo.getTargetClass(MixinInfo.java:1027) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinInfo.readTargetClasses(MixinInfo.java:1008) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinInfo.parseTargets(MixinInfo.java:896) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinConfig.prepareMixins(MixinConfig.java:869) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinConfig.prepare(MixinConfig.java:777) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.prepareConfigs(MixinProcessor.java:540) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.select(MixinProcessor.java:462) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.checkSelect(MixinProcessor.java:438) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:290) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:413) ~[fabric-loader-0.14.6.jar:?]
	... 7 more
```

**Comment by Derailedzack** [comment]
This version of the mod doesn't work with 1.18.2. This version of the mod is compatible with 1.18 and 1.18.1 apparently.

**Comment by Pyrofab** [comment]
This should be fixed by installing beta 9 for minecraft 1.18.2, or beta 11 for minecraft 1.19.
